helm/fiaas-deploy-daemon/templates/deployment.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  labels:
+    {{ include "fiaas-deploy-daemon.labels" . | nindent 4 }}
+    {{ include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.deployment.labels | indent 4 }}
+{{- if or .Values.annotations.global .Values.deployment.annotations }}
+  annotations:
+    {{- include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.annotations.global | indent 4 }}
+    {{- include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.deployment.annotations | indent 4 }}
+{{- end }}
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+    type: RollingUpdate
```

Comment:
What's the advantage of using RollingUpdate over Recreate as the strategy?

Reply:
There is no technical reason: I initially used Recreate when creating the chart, but changed it to RollingUpdate only because that's how fiaas-deploy-daemon deploys itself now, and I thought that keeping it like that would be one less change in this PR. It is perhaps more "correct" to use Recreate, since having more than one fiaas-deploy-daemon running in a namespace is not a good idea as that could for example cause duplicate deploys. What do you think?

Reply:
I think it would be a small improvement in the chart, and i can't see how it would negatively affect anything to change it to Recreate here.